src/resolve/resolved-dnstls-openssl.c

```diff
@@ -392,8 +392,8 @@ int dnstls_manager_init(Manager *manager) {
 
```

Comment:
Could you line break the commit message? It is too long per line.

Reply:
Ok